etc/firefox.profile

```diff
@@ -28,5 +28,11 @@ include whitelist-usr-share-common.inc
 # private-etc must first be enabled in firefox-common.profile
 #private-etc firefox
 
+dbus-user filter
+dbus-user.own org.mozilla.firefox.*
+dbus-user.talk org.freedesktop.Notifications
+dbus-system none
+ignore nodbus
```

Comment:
I get why we need to have `ignore nodbus` here. Does that mean we can remove it from firefox-common-addons.inc?

Reply:
Now needs to be `ignore dbus-user none`.

firefox-common.profile is used by firefox, thunderbird, waterfox, ... and disables D-Bus access. Of course we can't add the own rule to firefox-common. Because some program throw errors messages if D-Bus isn't available, but hardfail on D-Bus error due to work policy, I added this for now only to firefox.